Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp

```diff
@@ -2364,6 +2369,19 @@ ByteCodeParser::CallOptimizationResult ByteCodeParser::handleInlining(
         }
         data.cases.append(SwitchCase(m_graph.freeze(thingToCaseOn), calleeEntryBlock));
         VERBOSE_LOG("Finished optimizing ", callLinkStatus[i], " at ", currentCodeOrigin(), ".\n");
+
+        // Boosting inlining balance a bit for polymorphic calls. But we do not want to increase inliningBalance directly since it can be exhausted for one call.
+        // We refill balance when it is used in the other inlining. And the amount we refill is capped with polyInliningAdjustment.
+        if (inliningBalance < originalInliningBalance) {
+            unsigned usedBudget = inliningBalance - originalInliningBalance;
```

Comment:
Should this be `unsigned usedBudget = originalInliningBalance - inliningBalance;`?

Reply:
Oops, fixed.